remote-api/src/test/java/org/alfresco/rest/api/tests/NodeApiTest.java

```diff
@@ -6366,5 +6366,92 @@ public void testRequestRenditionContentDirectUrl() throws Exception
 
         HttpResponse dauResponse = post(getRequestContentDirectUrl(contentNodeId), null, null, null, null, 501);
     }
+
+    @Test
+    public void testRequestDeleteRendition() throws Exception
+    {
+        setRequestContext(user1);
```

Comment:
not keen on using a deprecated method, although this is already used throughout the code, so unless there is an obvious replacement it's probably ok to leave it for now